pkg/javadb/client.go

```diff
@@ -29,6 +29,9 @@ const (
 var (
 	// GitHub Container Registry
 	DefaultGHCRRepository = fmt.Sprintf("%s:%d", "ghcr.io/aquasecurity/trivy-java-db", SchemaVersion)
+
+	// GHC mirrors
```

Comment:
nit

```suggestion
	// GCR mirrors
```

Reply:
thanks! fixed in https://github.com/aquasecurity/trivy/pull/7679/commits/272b1a04ff23dcba2874d2835a4c2cc8b1ab3e15